### PR TITLE
add revision history for 0.590

### PR DESCRIPTION
**Pull request by gvanrossum**

### CODE REVIEW

**Comment by gvanrossum** [comment]
FWIW I've double-checked that this builds with Sphinx and looks fine.

File: docs/source/revision_history.rst [hunk 1]
```diff
@@ -3,13 +3,28 @@ Revision history
 
 List of major changes:
 
+- April 2018
+    * Publish ``mypy`` version 0.590 on PyPI.
+
+      * Document PEP 561 support.
```
Comment (JukkaL) [inline docs/source/revision_history.rst]: Add link to the relevant section in the docs.

File: docs/source/revision_history.rst [hunk 2]
```diff
@@ -3,13 +3,28 @@ Revision history
...
+
+      * Made incremental mode the default.
+
+      * Document ``--always-true`` and ``--always-false``.
```
Comment (JukkaL) [inline docs/source/revision_history.rst]: Link to the relevant docs?

File: docs/source/revision_history.rst [hunk 3]
```diff
@@ -3,13 +3,28 @@ Revision history
...
+
+      * Document ``follow_imports_for_stubs``.
```
Comment (JukkaL) [inline docs/source/revision_history.rst]: Link to the relevant docs?

File: docs/source/revision_history.rst [hunk 4]
```diff
@@ -3,13 +3,28 @@ Revision history
...
+
+      * Add coroutines to Python 3 cheat sheet.
+
+      * Add ``None`` return/strict-optional to common issues.
```
Comment (JukkaL) [inline docs/source/revision_history.rst]: Link to the relevant section?

**Pull request merged**